docs/usage.md

```diff
@@ -8,6 +8,13 @@ Refer to the table of [template blocks](./docs/template-blocks.md) and their def
 
 The [govuk_template sets a skip link](https://github.com/alphagov/govuk_template/blob/master/source/views/layouts/govuk_template.html.erb#L64-L68) to `#content`, but doesn't provide an element with `id="content"`. You'll need to add `id="content"` to your main content area, to ensure the skip link will work.
 
+    <main id="content" role="main" tabindex="-1">
```

Comment:
We provide the [HTML5Shiv bundled with the govuk_template JS](https://github.com/alphagov/govuk_template/blob/master/source/assets/javascripts/vendor/html5shiv.js) and have added `display:block`for main to the [govuk_template css](https://github.com/alphagov/govuk_template/blob/master/source/assets/stylesheets/_basic.scss#L16).